### ISSUE TITLE
BURNVEG2AIRC produces just one value

### ISSUE THREAD

**Issue by valeriabriones** [uaf-arctic-eco-modeling/dvm-dos-tem#619]
When outputting BURNVEG2AIRC at a monthly timestep at FRI of 50 years, it produces just one monthly value for the output. Would expect more than just one value for this output if it is being output with a FRI disturbance?
![MD1_fri50_burnveg2airc_monthly](https://github.com/uaf-arctic-eco-modeling/dvm-dos-tem/assets/90799683/0fa29d28-0cab-456f-959e-8c4ca850a06e)

**Comment by rarutter** [comment]
Yeah, it should burn every 50 years (unless the ecosystem crashed entirely). I'll take a look and see what's going on.

**Comment by rarutter** [comment]
Using the Toolik demo set, FRI 70, severity 3, master branch (2e4334c3) and the following run command, this is what I get.
`develop@7776206c1eff:/work$ ./dvmdostem -l fatal -p 100 -e 300 -s 0 -t 0 -n 0`.
It could be something different between Toolik and MD, so I'll try to replicate your run more closely later.
![BURNVEG2AIRC_FRI70](https://github.com/uaf-arctic-eco-modeling/dvm-dos-tem/assets/6707880/e298008d-ec05-43c4-8a9c-5d81987e5a1d)

**Comment by valeriabriones** [comment]
@rarutter ok. I’ll dissect a bit further and try a few different runs to see if it’s on my end. Thank you!

**Comment by valeriabriones** [comment]
@rarutter I was able to replicate your results using Toolik. It seems there is an issue with CoefA and CoefB at Murphy Dome. I can't recall from previous conversations, but was there a line of code to update these coefficients?
![Screenshot (79)](https://github.com/uaf-arctic-eco-modeling/dvm-dos-tem/assets/90799683/064d0ec8-b79b-4dbc-ab4b-c25d4c4834c7)

**Comment by rarutter** [comment]
@valeriabriones Those coefficients are actually parameters, set in `cmt_dimground.txt`. I'm concerned about the line where carbon in a PFT compartment is NaN - that may indicate an additional problem somewhere.
